### PR TITLE
[Test] Add qwen3-omni tests for audio_in_video and one word prompt

### PR DESCRIPTION
**Pull request by yenuo26**
PLEASE FILL IN THE PR DESCRIPTION HERE ENSURING ALL CHECKLIST ITEMS (AT THE BOTTOM) HAVE BEEN CONSIDERED.

## Purpose
Add qwen3-omni tests for audio_in_video and one word prompt
## Test Plan
1.run in local env
```
/workspace/.venv/bin/python -m pytest -s -v tests/e2e/online_serving/test_qwen3_omni_expansion.py -k "test_audio_in_video_001"  -m "advanced_model" --run-level "advanced_model"
```
2.run in ci
## Test Result
1.local
```
==================================================================== 1 passed, 18 warnings in 132.98s (0:02:12) ====================================================================
```
2.ci
bc2d3920-677e-4a33-acf9-e1310e6a7724

---
<details>
<summary> Essential Elements of an Effective PR Description Checklist </summary>

- [ ] The purpose of the PR, such as "Fix some issue (link existing issues this PR will resolve)".
- [ ] The test plan. Please provide the test scripts & test commands. Please state the reasons if your codes don't require additional test scripts. For test file guidelines, please check the [test style doc](https://docs.vllm.ai/projects/vllm-omni/en/latest/contributing/ci/tests_style/)
- [ ] The test results. Please paste the results comparison before and after, or the e2e results.
- [ ] (Optional) The necessary documentation update, such as updating `supported_models.md` and `examples` for a new model. **Please run `mkdocs serve` to sync the documentation editions to `./docs`.**
- [ ] (Optional) Release notes update. If your change is user-facing, please update the release notes draft.
</details>

**BEFORE SUBMITTING, PLEASE READ <https://github.com/vllm-project/vllm-omni/blob/main/CONTRIBUTING.md>** (anything written below this line will be removed by GitHub Actions)

### CODE REVIEW

**Comment by yenuo26** [comment]
@Shirley125 @amy-why-3459 PTAL

**Comment by amy-why-3459** [comment]
Please add a multi-concurrency precision test case for audio_in_video.

**Comment by yenuo26** [comment]
> Please add a multi-concurrency precision test case for audio_in_video.

done

**Comment by yenuo26** [comment]
@Gaohan123 @tzhouam please help to add a ready label

File: .buildkite/test-ready.yml [hunk 1]
```diff
@@ -183,44 +183,45 @@ steps:
...
-  #                 path: /mnt/hf-cache
-  #                 type: DirectoryOrCreate
+  - label: "Omni Model Test with H100"
+    depends_on: upload-ready-pipeline
+    commands:
+      - |
+        timeout 20m bash -c '
+          export VLLM_WORKER_MULTIPROC_METHOD=spawn
+          export VLLM_TEST_CLEAN_GPU_MEMORY="1"
+          #pytest -s -v tests/e2e/online_serving/test_qwen3_omni.py -m "core_model" --run-level "core_model"
```
Comment (hsliuustc0106) [inline .buildkite/test-ready.yml]: This swaps the normal H100 omni test job over to `test_qwen3_omni_expansion.py` and even leaves a `for debug, will be removed before merging` note in the committed pipeline. That means the PR is changing repository CI coverage in a temporary/debug-only way instead of adding a stable test signal. Could you revert the debug pipeline override and keep the new coverage in a dedicated test/job if it needs CI coverage?

**Comment by hsliuustc0106** [comment]
## Review Summary

**Verdict**: ✅ LGTM (No Blockers)

Test-only PR for Qwen3-omni edge cases. Good coverage expansion!

### Test Quality Check

✅ **Test correctness**: Tests use standard `omni_server` + `openai_client` fixtures
✅ **Coverage**: Tests 3 scenarios:
  - `test_audio_in_video_001` - audio-in-video input
  - `test_audio_in_video_002` - different audio-in-video config
  - `test_one_word_prompt_001` - minimal text input edge case

✅ **Test evidence**: PR shows 1 passed in 132.98s

### Non-blocking Observations

**conftest.py**: Large refactoring (121 additions, 32 deletions) - test infrastructure improvements. Not reviewing in detail since it's test infrastructure.

**test-ready.yml**: Modified CI config (39 additions, 38 deletions) - appears to be test scheduling changes.

---

Ready to merge.

File: .buildkite/test-ready.yml (hunk 1 above)
Comment (yenuo26) [reply .buildkite/test-ready.yml]: removed

**Comment by Gaohan123** [comment]
Please fix docs

**Comment by yenuo26** [comment]
> Please fix docs

Actually, I didn't modify any document. I will retry it.

**Comment by Gaohan123** [comment]
@yenuo26 Please resolve conflicts

**Comment by yenuo26** [comment]
> @yenuo26 Please resolve conflicts
There are currently no conflicts to resolve.

File: tests/e2e/online_serving/test_qwen3_omni_expansion.py [hunk 2]
```diff
@@ -393,3 +411,85 @@ def test_mix_to_text_audio_001(omni_server, openai_client) -> None:
...
+        "model": omni_server.model,
+        "messages": messages,
+        "stream": True,
+        "use_audio_in_video": True,
+        "key_words": {"video": VIDEO_KEY, "audio": AUDIO_KEY + ["beep", "electronic"]},
+    }
+    openai_client.send_omni_request(request_config, request_num=get_max_batch_size())
+
+
+@pytest.mark.skip(reason="There is a known issue: https://github.com/vllm-project/vllm-omni/pull/2019")
```
Comment (amy-why-3459) [inline tests/e2e/online_serving/test_qwen3_omni_expansion.py]: PR #2019 has been merged. Please remove the skip option and test whether the test cases pass.

Comment (yenuo26) [reply tests/e2e/online_serving/test_qwen3_omni_expansion.py]: During local testing, I found that when called concurrently, this error is not resolved.

**Comment by yenuo26** [comment]
@Gaohan123 @david6666666 please help to add nightly-test

File: tests/e2e/online_serving/test_qwen3_omni_expansion.py (hunk 2 above)
Comment (amy-why-3459) [reply tests/e2e/online_serving/test_qwen3_omni_expansion.py]: OK, I will continue to follow up on this issue and will remove the skip flag once it is resolved.

**Comment by yenuo26** [comment]
Can this be merged? I've already run L2+L4. The vllm-omni CI failed because the Diffusion L4 performance test didn't pass. @Gaohan123

**Comment by yenuo26** [comment]
b38ebbd1-d69d-400e-bd9f-a16c7816facd

I just modify test case in "Omni Model Test with H100", and it is success in nightly test.
Please review whether this PR can be merged. @Gaohan123 @david6666666
